src/pages/publication/publication_page/PublicationPageSeries.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Publication } from '../../../types/publication.types';
+import LabelContentRowForPublicationPage from './LabelContentRowForPublicationPage';
+import { ReferenceType } from '../../../types/references.types';
+import { useTranslation } from 'react-i18next';
+
+interface PublicationPageSeriesProps {
+  publication: Publication;
+}
+
+const PublicationPageSeries: FC<PublicationPageSeriesProps> = ({ publication }) => {
+  const [series, setSeries] = useState();
+  const { t } = useTranslation('publication');
+
+  useEffect(() => {
+    publication.reference.type === ReferenceType.BOOK && setSeries(publication.reference?.book?.series);
+    publication.reference.type === ReferenceType.REPORT && setSeries(publication.reference?.report?.series);
+  }, [publication]);
```

Comment:
samme her som for pageIdentifiers

Reply:
Vi bestemte oss for å ikke bry oss om dette pr nå. Datamodellen kommer sannsynligvis med en helt annen struktur på disse datene